conans/client/installer.py

```diff
@@ -24,7 +24,8 @@
 from conans.model.user_info import UserInfo
 from conans.paths import BUILD_INFO, CONANINFO, RUN_LOG_NAME
 from conans.util.env_reader import get_env
-from conans.util.files import (clean_dirty, is_dirty, make_read_only, mkdir, rmdir, save, set_dirty)
+from conans.util.files import (clean_dirty, is_dirty, make_read_only, mkdir, rmdir, save, set_dirty,
```

Comment:
do you still need to import `set_dirty`/`clean_dirty`?

Reply:
Yes, at line 188 and 211. Both set_dirty and clan_dirty are not in same context. If I use dirty_context it will put that code in another condition.